neurolib/models/multimodel/builder/aln.py

```diff
@@ -373,6 +371,35 @@ def tau_lookup(self, y, current_mu, current_sigma):
             self.tau_transfer_function,
         )
 
+    def _get_current_sigma(self, I_syn_sigma_exc, I_syn_sigma_inh, exc_inp, inh_inp, J_exc_max, J_inh_max, ext_sigma):
```

Comment:
instead of having the whole definition of ALN dynamics in the `_derivatives()`, I pulled out some functions and created helpers.. this way, it's easier to alter ALN (e.g. with ACh neuromodulation, or multisynapse etc)